sugondat-chain/pallets/blobs/src/lib.rs

```diff
@@ -227,6 +227,23 @@ pub mod pallet {
             });
             Ok(().into())
         }
+
+        #[pallet::call_index(1)]
+        #[pallet::weight(0)]
+        pub fn update_parameter(
```

Comment:
This call seems extremely dangerous and broad. What's the purpose of it?

Reply:
Yes, this call is undoubtedly dangerous. What I would like to achieve is simply a way to allow changes to the parameters of BlobAdjustmentFee with an extrinsic. Before using it, some necessary changes would be assigning proper weights and instead of accepting the key as bytes, it might be better to accept an enum that encapsulates both the key and value to avoid potential incorrect storage insertions.

The current implementation is just an experiment to demonstrate how parameters could be changed by governance/sudo

Reply:
There is already `system_setStorage` for setting storage arbitrarily - we should use that in favor of introducing another call.

Reply:
Uh, that's true! This makes that call completely useless. I will delete it